### PR TITLE
i18n - Bhojpuri support

### PR DESCRIPTION
**Pull request by prathameshsnagawade**
What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)
Feature to add i18n - Bhojpuri Language Support

Other information:
-Kindly review and merge this PR

### CODE REVIEW

File: docs/docs/notification-center/react-components.md [hunk 1]
```diff
@@ -212,7 +212,8 @@ The `i18n` prop can accept 2 different types of values
           <li><code>tr</code> (Turkish)</li>
           <li><code>te</code> (Telugu)</li>
           <li><code>vi</code> (Vietnamese)</li>
-          
+          <li><code>bh</code> (Bhojpuri)</li>
```
Comment (p-fernandez) [inline docs/docs/notification-center/react-components.md:215]: Unfortunately there is no ISO code 639-1 for Bhojpuri. And currently is the ISO code we support. We would need to hold this one.

**Review by akshitagupta15june — COMMENTED** [review]
The language code i just saw was "bho"

Référence https://digital.library.unt.edu/explore/collections/BPLR/

**Comment by p-fernandez** [comment]
> The language code i just saw was "bho"
> 
> Référence https://digital.library.unt.edu/explore/collections/BPLR/

That's the language code for ISO 639-2 and 639-3. As you can see from our codebase and other language support PRs you have contributed with, we are using ISO 639-1 for language codes. That means 2 characters. We would need to study the implications on using other different ISOs to be able to support 3 characters language codes.

**Comment by akshitagupta15june** [comment]
> > The language code i just saw was "bho"
> > Référence https://digital.library.unt.edu/explore/collections/BPLR/
> 
> That's the language code for ISO 639-2 and 639-3. As you can see from our codebase and other language support PRs you have contributed with, we are using ISO 639-1 for language codes. That means 2 characters. We would need to study the implications on using other different ISOs to be able to support 3 characters language codes.

True, we need to test for 3 characters

**Comment by scopsy** [comment]
Thank you for you contribution @prathameshsnagawade  unfortuently I will have to close this issue for now due to our inability to support this ISO code right now. We might revisit it in the future in case we will fix this underlying need on our end. 🙏

**Pull request closed without merging**